package.json

```diff
@@ -74,10 +74,8 @@
     "temp": "^0.8.3",
     "webpack": "^2.1.0-beta.25"
   },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/yarnpkg/yarn.git"
-  },
+  "repository": "yarnpkg/yarn",
+  "bugs": "https://github.com/yarnpkg/yarn/issues",
```

Comment:
This is not needed. It's inferred from the `repository` field.

https://github.com/npm/normalize-package-data#what-normalization-currently-entails